lab-teddy/app/entry.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('angular')
+.module('ngAdventure', []);
+
+require('./service/map-service.js');
```

Comment:
Note that in class we are using require.context(). This actually helps because you wouldn't have to keep adding every new component you make to your entry file. I suggest using that in the future.